README.md

```diff
@@ -141,6 +141,7 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
 * [Data Science : R Basics](https://courses.edx.org/courses/course-v1:HarvardX+PH125.1x+2T2019): Course about R basics presented by Harvard University
 * [Kaggle: Intro to Machine Learning](https://www.kaggle.com/learn/intro-to-machine-learning): Learn the core ideas in machine learning, and build your first models.
 * [Data quest](https://https://www.dataquest.io/): From intro do advanced data science and analytics content in short lessons.
+* [Neural Networks from Scratch](https://www.youtube.com/watch?v=Wo5dMEP_BbI): A youtube playlist to learn neural networks from scratch using python
```

Comment:
This is only a 15 mn video, you can include the channel in the YouTube section if you want.

Reply:
The link was wrong, I have updated it with the right one. It's a playlist with 3 hrs content.